src/pages/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -20,9 +20,21 @@ import ARROW from "assets/images/admin/dashboard-arrow.svg";
 import "./WorldsDashboard.scss";
 
 export const WorldsDashboard: React.FC = () => {
-  const user = useUser();
+  const { ownedVenues } = useOwnedVenues({});
 
-  const worlds = useOwnWorlds(user.userId);
+  const ownedUniqueWorldIds = useMemo(
+    () => [...new Set(ownedVenues.map((venue) => venue.worldId))],
+    [ownedVenues]
+  );
+
+  const worlds = useWorlds();
+
+  // NOTE: We could do this by forming a firebase query where([world.id, 'in', ownedUniqueWorldIds]), but it has a limit of 10 items in ownedUniqueWorldIds.
```

Comment:
Or, by doing `world.owners array-contains user.id`

Reply:
I'm happy with how you've done the filtering in the frontend btw

Reply:
Any of these queries would still result in a hard limit of 10 entries. I wanted to avoid it

Reply:
There isn't a hard limit of 10 when using `array-contains`. There is a hard limit of 10 when using `array-contains-any`.
Docs here: https://firebase.google.com/docs/firestore/query-data/queries#limitations_2

Just confirmed by creating a document with an array of 12 things and querying it with `array-contains`

Reply:
No need to fix anything here. Just an FYI